challenge_help_me_carry/src/challenge_help_me_carry.py

```diff
@@ -27,11 +28,6 @@ def __init__(self, robot):
 
         self.car_waypoint = ds.EntityByIdDesignator(robot, id=challenge_knowledge.waypoint_car['id'])
 
-        self.place_position = ds.LockingDesignator(ds.EmptySpotDesignator(robot, self.target_destination,
-                                                                          name="placement",
-                                                                          area=challenge_knowledge.default_area),
-                                                   name="place_position")
-
         self.empty_arm_designator = ds.UnoccupiedArmDesignator(robot, {}, name="empty_arm_designator")
 
         # With the empty_arm_designator locked, it will ALWAYS resolve to the same arm, unless it is unlocked.
```

Comment:
Please also move these lines

Reply:
Why? these lines are about the empty arm designator. The new API of the emptyspotdesignator requires an arm so I just moved it to after the definition of the arm_designator

Reply:
That sounds good!